packages/core/theme/src/components/slider.ts

```diff
@@ -0,0 +1,134 @@
+import type {VariantProps} from "tailwind-variants";
+
+import {dataFocusVisibleClasses} from "../utils";
+import {tv} from "../utils/tv";
+
+/**
+ * Slider wrapper **Tailwind Variants** component
+ *
+ * @example
+ *
+ * TODO: Add example here
+ * const styles = slider()
+ *
+ * <span className={styles} />
+ */
+const slider = tv({
+  slots: {
+    base: "flex flex-col w-full gap-1",
+    labelWrapper: "w-full flex justify-between",
+    label: "",
+    output: "",
+    track: [
+      "flex",
+      "w-full",
+      "relative",
+      "rounded-full",
+      "bg-default-300/50",
+      "border-transparent",
+    ],
+    filler: "h-full absolute",
+    thumb: [
+      "rounded-full",
+      "top-1/2",
+      "flex",
+      "justify-center",
+      "items-center",
+      "before:absolute",
+      "before:w-11",
+      "before:h-11",
+      "before:rounded-full",
+      "data-[dragging=true]:after:scale-80",
```

Comment:
What about something like this?
```suggestion
      "data-[dragging=true]:after:scale-80",
      "data-[focus-visible=true]:after:scale-80",
```

Reply:
I think when navigating with the keyboard the interaction/animations should be as simple as possible